src/config.ts

```diff
@@ -197,35 +187,58 @@ export class NodeInfo {
   public RpcUrl(): string {
     return `${this.host}:${this.rpcPort}`;
   }
+
+  public async PreloadEnded(): Promise<boolean> {
+    const resp = await axios.post(
+      `http://${this.GraphqlServer()}`,
+      {
+        query: "query{nodeStatus{preloadEnded}}",
+      },
+      {
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    );
```

Comment:
Just a note but we can reuse `graphql-request` module here since we already have it as a dependency.